resources/views/header-script.blade.php

```diff
@@ -16,7 +16,11 @@
             window.dataLayer.push({!! $item->toJson() !!});
         @endforeach
 
-        window.addEventListener('push_to_data_layer', (event) => window.dataLayer.push(event.detail))
+        @if(settings (\Astrogoat\Elevar\ElevarSettings::class)-›isEnabled())
```

Comment:
It looks like where's something up with the arrow characters...

```suggestion
        @if(settings (\Astrogoat\Elevar\ElevarSettings::class)->isEnabled())
```